app/views/candidate_mailer/eoc_deadline_reminder.erb

```diff
@@ -0,0 +1,18 @@
+Dear <%= @application_form.first_name %>,
+
+# Complete your teacher training application – courses fill up quickly at this time of year
+
+Submit your application as soon as you can to get on a course starting in the <%= RecruitmentCycle.current_year %> to <%= RecruitmentCycle.next_year %> academic year:
+
+<%= candidate_magic_link(@application_form.candidate) %>
+
```

Comment:
```
if @application_form.apply_1?
<% "The deadline to submit your application is 6pm on #{CycleTimetable.apply_1_deadline.to_s(:govuk_date)} but courses may fill up before then." %>
else 
<% "The deadline to submit your application is 6pm on #{CycleTimetable.apply_2_deadline.to_s(:govuk_date)} but courses may fill up before then." %>

```

Reply:
Having the logic like this is what caused the line break problem

https://github.com/DFE-Digital/apply-for-teacher-training/pull/5039#issuecomment-869631873

Reply:
ah right okay. all good then